### ISSUE TITLE
Allow nil values

### ISSUE THREAD

**Issue by sparkertime** [blakesmith/pghstore-clj#3]
Right now, to-hstore and from-hstore can only be called on non-nil values. It's a bit annoying to have to nil-check the value before calling the methods - what would you say to a patch to extends the *Storable types for nil?

**Comment by blakesmith** [comment]
What would nil store as? Empty string? Can the hstore value be null?

**Comment by sparkertime** [comment]
Yeah, I would think that (to-pghstore nil) -> nil. They can indeed be null.

**Comment by blakesmith** [comment]
Cool, I'd definitely take a patch for this.
